std/process.d

```diff
@@ -3233,11 +3242,15 @@ private:
     assertThrown(environment["std_process"]);
 
     // get() without default value
-    assert (environment.get("std_process") == null);
+    assert (environment.get("std_process") is null);
 
     // get() with default value
     assert (environment.get("std_process", "baz") == "baz");
 
+    // get() on an empty (but present) value
+    environment["std_process"] = "";
```

Comment:
I don't think it's the wine issue (I'm not sure of the test systems, but I'm quite certain they are not wine).

But put the version block back, because this is a different bug. I don't quite understand what the unittest that fails is doing, but it's self contained. In effect, if someone had something in their environment that mimicked what you did, then it would still fail.

We need another bug report for it. I'll have to figure it out when I get a chance.